django_app/settings.py

```diff
@@ -110,7 +110,6 @@
         },
         # to capture email/profile data from google Oauth
         "SCOPE": [
-            "profile",
```

Comment:
Again. Why don't you break this stuff into separate PRs that do separate things?


If you submitted this one-liner as a PR I'd approve it immediately. But the rest of it needs some changes. Why not make multiple independent PRs like we've talked about several times?

Reply:
Ok,
From now on I will make PRs further small